os/include/debug.h

```diff
@@ -499,7 +499,7 @@ int get_errno(void);
 #define mllvdbg(...)
```

Comment:
Please change "This patch is for change in mfdbg to use lldbg instead of dbg in flat build" to "This patch is to change mfdbg to use lldbg instead of dbg in flat build"